### ISSUE TITLE
install on lab 0.29.1 is failing ... complains about Conflicting Dependencies

### ISSUE THREAD

**Issue by Analect** [jupyterlab/jupyterlab-google-drive#101]
@ian-r-rose 
Just a heads up.

```
Step 30/39 : RUN jupyter labextension install @jupyterlab/google-drive
 ---> Running in 251d10ac05f0
> node node-version-check.js
> npm pack @jupyterlab/google-drive
jupyterlab-google-drive-0.7.0.tgz
Traceback (most recent call last):
  File "/opt/conda/bin/jupyter-labextension", line 6, in <module>
    sys.exit(jupyterlab.labextensions.main())
  File "/opt/conda/lib/python3.6/site-packages/jupyter_core/application.py", line 266, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "/opt/conda/lib/python3.6/site-packages/traitlets/config/application.py", line 658, in launch_instance
    app.start()
  File "/opt/conda/lib/python3.6/site-packages/jupyterlab/labextensions.py", line 174, in start
    super(LabExtensionApp, self).start()
  File "/opt/conda/lib/python3.6/site-packages/jupyter_core/application.py", line 255, in start
    self.subapp.start()
  File "/opt/conda/lib/python3.6/site-packages/jupyterlab/labextensions.py", line 64, in start
    for arg in self.extra_args]
  File "/opt/conda/lib/python3.6/site-packages/jupyterlab/labextensions.py", line 64, in <listcomp>
    for arg in self.extra_args]
  File "/opt/conda/lib/python3.6/site-packages/jupyterlab/commands.py", line 137, in install_extension
    return IOLoop.instance().run_sync(func)
  File "/opt/conda/lib/python3.6/site-packages/tornado/ioloop.py", line 458, in run_sync
    return future_cell[0].result()
  File "/opt/conda/lib/python3.6/site-packages/tornado/concurrent.py", line 238, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 4, in raise_exc_info
  File "/opt/conda/lib/python3.6/site-packages/tornado/gen.py", line 1069, in run
    yielded = self.gen.send(value)
  File "/opt/conda/lib/python3.6/site-packages/jupyterlab/commands.py", line 208, in install_extension_async
    raise ValueError(msg)
ValueError: 
"@jupyterlab/google-drive@0.7.0" is not compatible with the current JupyterLab
Conflicting Dependencies:
JupyterLab              Extension            Package
>=0.12.0-0 <0.13.0-0    >=0.11.0-0 <0.12.0-0 @jupyterlab/application
>=0.12.4-0 <0.13.0-0    >=0.11.0-0 <0.12.0-0 @jupyterlab/apputils
>=0.12.0-0 <0.13.0-0    >=0.11.0-0 <0.12.0-0 @jupyterlab/coreutils
>=0.12.0-0 <0.13.0-0    >=0.11.0-0 <0.12.0-0 @jupyterlab/docmanager
>=0.12.0-0 <0.13.0-0    >=0.11.0-0 <0.12.0-0 @jupyterlab/filebrowser
>=0.12.0-0 <0.13.0-0    >=0.11.0-0 <0.12.0-0 @jupyterlab/rendermime
>=0.51.0-0 <0.52.0-0    >=0.50.0-0 <0.51.0-0 @jupyterlab/services

```

**Comment by ian-r-rose** [comment]
Yep, I am travelling right now, but plan to have an update this weekend.

**Comment by krokha72** [comment]
Ian, I have same issue reinstalling jupyterlab-google-drive-0.7.0.tgz extension for jupyter lab ver 0.29.1
the   google-drive install output is same as Analect have provided
Thanks
Alex

**Comment by ian-r-rose** [comment]
I have just published version `v0.8.0`, for JupyterLab 0.29, give it a spin!

**Comment by krokha72** [comment]
I will, thanks a lot for your help

On Nov 12, 2017 2:30 AM, "Ian Rose" <notifications@github.com> wrote:

I have just published version v0.8.0, for JupyterLab 0.29, give it a spin!

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/jupyterlab/jupyterlab-google-drive/issues/101#issuecomment-343709024>,
or mute the thread

**Comment by krokha72** [comment]
New build worked, tx a lot
